### PR TITLE
HW-Week1-Ramzi

### PR DESCRIPTION
**Pull request by ramziash**

### CODE REVIEW

File: Week1/homework/e2/e2.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  database : 'world'
+});
+connection.connect();
+
+
+
+
+// Q1 - greater than 8mil
+
+let q1 = "select name,population from country where population > 8000000 order by population "
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q1 answer ", results);
+});
+
+
+// Q2 - land in name
+
+let q2 = "select name from country where name like '%land%'"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q2 answer ", results);
+});
+
+
+// Q3 - population between 500K & 1Mil
+
+let q3 = "select name from country where population between 500000 and 1000000"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q3 answer ", results);
+});
+
+
+// Q4 - countries in the EU
+
+let q4 = "select name from country where continent = 'europe'"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q4 answer ", results);
+});
+
+
+// Q5 - descending order of their surface areas.
+
+let q5 = "select name from country order by surfacearea desc"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+connection.query(q5, function (error, results, fields) {
+    if (error) {
+        throw error;
+    }
+    console.log("Q5 answer ", results);
+});
+
+
+// Q6 - names of all the cities in the Netherlands?
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q6 answer ", results);
+});
+
+
+// Q7 - population of Rotterdam?
+
+let q7 = "select population from city where name ='rotterdam'"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q7 answer ", results);
+});
+
+
+// Q8 -  top 10 countries by Surface Area
+
+let q8 = "select name from country order by surfacearea desc limit 10"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q8 answer ", results);
+});
+
+
+// Q9 - top 10 most populated cities?
+
+let q9 = "select name from city order by population desc limit 10"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e2/e2.js [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+    if (error) {
+        throw error;
+    }
+    console.log("Q9 answer ", results);
+});
+
+
+// Q10 - population number of the world?
+
+let q10 = "select sum(population) from country"
```
Comment (roxana-lafuente) [inline Week1/homework/e2/e2.js]: Query OK!

File: Week1/homework/e1/e1.js [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+var connection = mysql.createConnection({
+    host     : 'localhost',
+    user     : 'hyfuser',
+    password : 'hyfpassword',
+    database : `${dbName}`
+  });
+  
+
+const create_tables_query =[
+    "create table Invitee (invitee_no int not null, invitee_name varchar(50) not null, invited_by varchar(50))",
```
[the remark this answers is not in the document]
Comment (ramziash) [reply Week1/homework/e1/e1.js]: this is the one shared.
https://github.com/HackYourFuture/databases/blob/master/Week1/MAKEME.md

**Pull request closed without merging**